tests/models/gemma/test_modeling_gemma.py

```diff
@@ -815,7 +824,7 @@ def test_model_7b_4bit(self):
         EXPECTED_TEXTS = {
             7: [
                 "Hello I am doing a project for my school and I am trying to make a program that will take a number and then",
-                """Hi today I am going to talk about the new update for the game called "The new update" and I""",
+                "Hi today I am going to talk about the best way to get rid of acne. miniaturing is a very",
```

Comment:
`test_model_7b_4bit` starts to fail since April 24. I still get failures when I checkout to commits before that date. Likely some changes in the bnb. Let's just update.

Reply:
(well, it probably fails even in earlier date as we switch slack report channels a few month back and I only check one channel that is currently used)